react-native/react-native.d.ts

```diff
@@ -3354,7 +3386,99 @@ declare namespace  __React {
     }
 
     export interface StyleSheetStatic extends React.ComponentClass<StyleSheetProperties> {
-        create<T>( styles: T ): T;
```

Comment:
@bgrieder I'm having second thought on this. Maybe we should keep it this way.

The actual implementation is
```
 create<T: Object, U>(obj: T): {[key:$Keys<T>]: number}
```
https://github.com/facebook/react-native/blob/master/Libraries/StyleSheet/StyleSheet.js#L163


But it seems so difficult to comprehend and use.